vanilla-framework/vanilla-framework
# 2055 merge code examples

## Done

Merged the docs for all code components

## QA

- Pull code
- cd to /docs
- Run `./run serve --watch`
- Open http://localhost:8104/en/
- See that there is only "Code" under the "Text" heading on the page
- Click the "code" link and view the page

## Details
Fixes #2055
